README.md

```diff
@@ -4,4 +4,27 @@
 [![codecov](https://codecov.io/gh/notaryproject/notation-core-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-core-go)
 [![Go Reference](https://pkg.go.dev/badge/github.com/notaryproject/notation-core-go.svg)](https://pkg.go.dev/github.com/notaryproject/notation-core-go@main)
 
-Contains support for Notary Project signature envelope, and format specific implementation
+notation-core-go implements signature generation, parsing and revocation related functionalities based on the [Notary Project specifications](https://github.com/notaryproject/specifications).
```

Comment:
nit:
notation-core-go also provides `certificate and certificate chain validation`.